backend/fms_core/resources/_generic.py

```diff
@@ -2,14 +2,40 @@
 
 from import_export import resources
 from import_export.results import RowResult
+from django.core.exceptions import ValidationError
 from reversion.models import Version
 
 
 class GenericResource(resources.ModelResource):
-    clean_model_instances = True
-    skip_unchanged = True
     ERROR_CUTOFF = 20
```

Comment:
Would also change the variable name to something like: MAX_NB_ERRORS_ALLOWED or something like this, which would be more explicit imo